plugins/aws/neptune/neptuneDBMinorVersionUpgrade.js

```diff
@@ -0,0 +1,55 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Neptune Database Minor Version Upgrade',
+    category: 'Neptune',
+    domain: 'Databases',
+    severity: 'Medium',
+    description: 'Ensures Auto Minor Version Upgrade is enabled on Neptune database instances.',
+    more_info: 'Neptune Database supports automatically upgrading the minor version of the database, which should be enabled to ensure security fixes are quickly deployed.',
```

Comment:
can you make it more descriptive? Maybe something like. 
AWS Neptune database service releases engine version upgrades regularly to introduce software features, bug fixes, security patches and performance improvements. Enabling this feature ensures that minor engine upgrades are applied automatically to the instance during the maintenance window.